### PR TITLE
Semi Transparent shader implementation

### PR DESCRIPTION
**Pull request by ezanageressu**

### CODE REVIEW

File: modules/core/ShaderSSLT.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+namespace rt {
+	/**
+		 * @brief Subsurface light transport shader
+		 */
+	class CShaderSSLT : public CShaderFlat {
+	private:
+		CScene& m_scene;
+		const Vec3f exitColor = RGB(0.4f, 0.4f, 0.4f);
+		const Vec3f	m_color = Vec3f::all(0.5f);
+
```
Comment (ereator) [inline modules/core/ShaderSSLT.h:16]: I think it makes sense to add one more member-variable `m_opacity` here and initialize in the constructor as an argument

File: modules/core/ShaderSSLT.cpp [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+namespace rt
+{
+	Vec3f CShaderSSLT::shade(const Ray& ray) const
+	{
+		Vec3f res = Vec3f::all(0);
+
+		Vec3f color = m_color;
+
+		Ray I(ray.hitPoint(), ray.dir, ray.counter + 1);
+		res = color + reTrace(I);
```
Comment (ereator) [inline modules/core/ShaderSSLT.cpp]: This one makes the object to glow: you add light here. The better choice would be to mix the color of the object and the color of the re-traced ray via a linear combination alpha * color + beta * reTrace(I), where alpha + beta = 
For the coefficients we can use the `m_opacity` variable , e.g. as follows:
`res = m_opacity * color + (1.0f - m_opacity) * reTrace(I);`

File: modules/core/ShaderSSLT.h [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+#include "Scene.h"
+
+namespace rt {
+	/**
+		 * @brief Subsurface light transport shader
+		 */
+	class CShaderSSLT : public CShaderFlat {
+	private:
+		CScene& m_scene;
+		const Vec3f exitColor = RGB(0.4f, 0.4f, 0.4f);
```
Comment (ereator) [inline modules/core/ShaderSSLT.h]: I would add prefix `m_` here as this is a member-variable

**Review by ereator — CHANGES_REQUESTED** [review]
Just a small modification is needed, otherwise is good!

**Comment by ezanageressu** [comment]
I have applied the changes!

**Pull request merged**